home/templates/home/email/contributors-deadline-reminder.txt

```diff
@@ -101,7 +101,11 @@ You can edit your final application(s) until {{ current_round.contributions_clos
 
 {{ request.scheme }}://{{ request.get_host }}{% url 'dashboard' %}
 
-The final application will ask you to fill out a timeline for the internship project. If you have trouble with this, or a mentor isn't responsive to your questions about the timeline, do the best you can. You should submit a final application by the final application deadline, even if you feel the timeline isn't complete or perfect.
+The final application will ask you to fill out a timeline for the internship project. For guidance on how to prepare your project timeline, please refer to our documentation.
+
+{{ request.scheme }}://{{ request.get_host }}{% url 'docs-applicant' %}#final-application
+
+If you still have trouble with this, or a mentor isn't responsive to your questions about the timeline, do the best you can. You should submit a final application by the final application deadline, even if you feel the timeline isn't complete or perfect.
```

Comment:
Thinking about expanding this change to something like this:

```suggestion
Your project timeline doesn't need to be perfect, but it should be flexible enough to accommodate any changes in plans and time commitments, or unexpected blockers. Please submit your final application even if your project timeline isn't complete — if you're selected as an intern, you will have the chance to review it alongside your mentors.
```

Reply:
This is perfect!